package.nls.json

```diff
@@ -0,0 +1,246 @@
+{
+    "vscode-docker.description": "Makes it easy to create, manage, and debug containerized applications.",
```

Comment:
Need to validate that there are no duplicate IDs, here and in the source files themselves.

Reply:
Yeah, there is a definite need for a NLS extension to help with these sorts of editing and validation tasks; it's still very much a mysterious process to me.  Maybe if we can find a way to tie that to containers, management will let us work on one.  :-)

Reply:
Validated and found two duplicate IDs, now fixed.